roles/check_reboot/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# tasks file for check_reboot
+- name: check if reboot is needed
+  stat:
+    path: /var/run/reboot-required
+  register: check_reboot
```

Comment:
Did any of our existing boxes include this? Wondering if any of our machines need a reboot right now or not.